tasks/broad/ImputationBeagleQcTasks.wdl

```diff
@@ -37,24 +38,26 @@ task QcChecks {
         if grep -q "^##fileformat=VCFv[4-9]\." header.vcf; then
             echo "VCF version 4.0+";
         else
-            echo "VCF version < 4.0 or not found;" >> qc_messages.txt;
+            echo "VCF version < 4.0 or not found." >> qc_messages.txt;
```

Comment:
I changed all the error message ending punctuation from semicolons to colons. That way the output doesn't need to be modified to look good. (Currently the service strips a trailing colon if it exists; this will be backwards compatible and that can be removed in future)

Reply:
the service currently doesn't replace the semicolons, we just trim the last one. so currently the service converts `This is problem one; This is problem two;` to `This is problem one; This is problem two`
and with my change, the service can just return the full string `This is problem one. This is problem two.` without changing it

Reply:
perfect thanks for the explanation!